### PR TITLE
Improvements and cleanups

### PR DESCRIPTION
**Pull request by williamdes**
- [ ] Remove gulp
- [x] Remove bower
- [x] Add .editorconfig
- [x] Add yarn.lock file
- [x] Translate README.md
- [x] Boot the project
- [x] Refresh docker version
- [ ] Be sure that all the building and installing process works
- [ ] Make tests work

Ref: https://github.com/code-lts/welcome/issues/5
Ref: #2

### CODE REVIEW

**Comment by williamdes** [comment]
My docker-compose version is running !
@svetlyak40wt I started moving commands to a Makefile to clean up the multiple ways of doings things to a very simple and robust way of running commands.

Either I am a total noob or there is duplication to clean:
- `.commands`
- `.tmuxp.yaml`
- `fabfile.py`
- `tasks.py`

**Comment by williamdes** [comment]
image

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+update_requirements:
+	pip-compile --annotate requirements.in
+	pip-compile --annotate requirements-dev.in
+
+SERVER_PORT=80
+run_server:
+	python ./manage.py runserver 0.0.0.0:${SERVER_PORT}
+
+migrate:
+	python ./manage.py migrate
```
Comment (svetlyak40wt) [inline Makefile:72]: You have to add all these commands to a `.PHONY` target:

https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html#Phony-Targets

**Comment by svetlyak40wt** [comment]
Great work!

File: Makefile (hunk 1 above)
Comment (williamdes) [reply Makefile:72]: Thank you, I will do that

Comment (williamdes) [reply Makefile:72]: Done as 6f2c0d75467222b5e076abc2351ffa48172d9f93

**Comment by williamdes** [comment]
I am currently unsure to keep up with making more changes on this project.
Would you merge my work ? (or part of it ?)

**Comment by svetlyak40wt** [comment]
This is ok William. Thank you for the help.

**Comment by williamdes** [comment]
> This is ok William. Thank you for the help.

Thank you for refreshing the project :)

**Pull request merged**